src/main/resources/swagger/api-docs.yaml

```diff
@@ -263,6 +263,59 @@ paths:
           description: Internal Error
           schema:
             $ref: '#/definitions/ErrorReport'
+  '/notebooks/{googleProject}/{clusterName}/api/localize':
+    post:
+      summary: Localize files to/from a Jupyter notebook server
+      description: |
+        Sends a command to a Jupyter notebook server to localize files between the server and GCS.
+        This operation will happen asynchronously; output, including any errors, will appear in `localization.log` in the working directory of the Jupyter notebook server.
+      operationId: proxyLocalize
+      tags:
+        - notebooks
+      parameters:
+        - in: path
+          name: googleProject
+          description: googleProject
+          required: true
+          type: string
+        - in: path
+          name: clusterName
+          description: clusterName
+          required: true
+          type: string
+        - in: body
+          description: |
+            JSON object. Keys represent destinations, values represent sources.
```

Comment:
Is there a reason you made the key the destination, and not the other way around? It seems a little counter-intuitive.

Reply:
You might want to put the same source file in multiple destinations, which you can't do with it the other way around.

I've had people tell me this way around is more intuitive so I guess it's all perspective :)

Reply:
Good point. Although this way you can't use the same destination directory multiple times.

Is there a python way to allow duplicate keys? E.g. like a `List[Tuple[String, String]]` instead of a `Map[String, String]`?